lib/src/test/resources/compileContracts.sh

```diff
@@ -8,16 +8,9 @@ readonly V1_SCARB_VERSION="0.4.0"
 readonly V2_SCARB_VERSION="0.7.0"
 readonly V2_6_SCARB_VERSION="2.6.0"
 
-# TODO: (#365) Use asdf action to install scarb instead
-echo "Installing scarb..."
-asdf plugin add scarb || true
-echo "Done!"
```

Comment:
This script is also used to compile contracts locally, on the machine, right? So I suppose we should have a check here for all the versions, to make sure they are indeed installed

Reply:
Correct, so if we want to leave compile scripts as it is i.e. checking and installing scarb versions, then issue's description isn't applicable:

> Can simplify compileContracts.sh script by installing scarb plugin on CI instead of using asdf action.

Reply:
we do not need to do that, just a check for valid versions should suffice. if those versions are not present we can just error out with an appropriate message.

Reply:
No need to add extra plugin/version checks here as `asdf` perform checks on its own.